src/components/bridge/ethereum/L1History.vue

```diff
@@ -0,0 +1,270 @@
+                  :href="
+                    getExplorerUrl(
+                      !checkIsL1(history.network_id) ? l1Network : l2Network,
+                      history.claim_tx_hash
+                    )
+                  "
+                  class="action-button link-button"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  <span>
+                    {{
+                      (!checkIsL1(history.network_id) ? l1Network : l2Network).replace('zkEVM', '')
+                    }}
+                  </span>
+                  <div class="container--explorer-icon">
+                    <astar-icon-external-link />
+                  </div>
+                </a>
+                <button
+                  v-else-if="history.isActionRequired"
+                  class="action-button link-button status--claim"
+                  @click="async () => await claim(history)"
+                >
+                  {{ $t('bridge.claim') }}
+                </button>
+                <div v-else class="link-button" />
+              </div>
+            </div>
+          </div>
+          <div v-else>
+            <q-skeleton class="skeleton--card" />
+          </div>
+        </div>
+      </div>
+    </div>
+    <div v-else>
+      <div v-if="isLoadingHistories">{{ $t('common.loading') }}</div>
+      <div v-else>{{ $t('bridge.noHistory') }}</div>
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { isHex } from '@polkadot/util';
+import { ethers } from 'ethers';
+import { DateTime } from 'luxon';
+import TokenBalance from 'src/components/common/TokenBalance.vue';
+import { EVM, blockExplorerUrls } from 'src/config/web3';
+import { useNetworkInfo } from 'src/hooks';
+import {
+  BridgeHistory,
+  EthBridgeNetworkName,
+  checkIsL1,
+  zkBridgeIcon,
+} from 'src/modules/zk-evm-bridge';
+import { PropType, defineComponent, ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+
+enum TxStatus {
+  Completed = 'completed',
+  InProgress = 'inProgress',
+}
+
+export default defineComponent({
+  components: { TokenBalance },
+  props: {
+    l1Network: {
+      type: String,
+      required: true,
+    },
+    l2Network: {
+      type: String,
+      required: true,
+    },
+    isLoadingHistories: {
+      type: Boolean,
+      required: true,
+    },
+    fetchUserHistory: {
+      type: Function,
+      required: true,
+    },
+    handleClaim: {
+      type: Function,
+      required: true,
+    },
+    histories: {
+      type: Object as PropType<BridgeHistory[]>,
+      required: true,
+    },
+  },
+  setup(props) {
+    const { nativeTokenSymbol } = useNetworkInfo();
+    const { t } = useI18n();
+
+    const syncIndex = ref<number | undefined>();
+
+    const handleUpdateHistory = async (index: number): Promise<void> => {
+      try {
+        syncIndex.value = index;
+        await props.fetchUserHistory();
+      } catch (error) {
+        console.error(error);
```

Comment:
Does this error needs to be show to an user?

Reply:
I think no need